CHANGELOG.asciidoc

```diff
@@ -20,6 +20,7 @@ endif::[]
 
 === Unreleased
 * Introduce *apm-agent-tracer* module that contains a minimal, zero-dependency `Tracer` API. - {pull}3043[#3043]
+* Move `GlobalTracer` to API module and substitute API where this is possible without further changes.
```

Comment:
We usually use the changelog just for changes which cause a visible effect to the end user. Your changes are just refactoring, so I think they shouldn't be part of the changelog.

Can you remove your two entries here?